extensions/amp-date-countdown/amp-date-countdown.md

```diff
@@ -97,6 +97,73 @@ This table provides examples of formatted values specified in a Mustache templat
 
 The experimental `1.0` version of `amp-date-countdown` uses the attribute name `count-up` instead of `data-count-up` as in `0.1` to support the "count up" feature. See the `count-up` section under `Attributes` below for more details.
 
+### Standalone use outside valid AMP documents (optional)
+
+
+Bento AMP allows you to use AMP components in non-AMP pages without needing to commit to fully valid AMP. You can take these components and place them in implementations with frameworks and CMSs that don't support AMP. Read more in our guide `[Use AMP components in non-AMP pages](https://amp.dev/documentation/guides-and-tutorials/start/bento_guide/)`.
+
+#### Example
+
+The example below demonstrates `amp-date-countdown` component in standalone use.
+
+[example preview="top-frame" playground="false"]
+```
+<head>
+...
+<script async src="https://cdn.ampproject.org/v0.js"></script>
+<link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/amp-date-countdown.css">
+<script async custom-element="amp-date-countdown" src="https://cdn.ampproject.org/v0/amp-date-countdown-1.0.js"></script>
+...
+</head>
+<amp-date-countdown
+  id="my-date-countdown"
+  timestamp-seconds="2147483648"
+  layout="fixed-height"
+  height="50">
+  <template type="amp-mustache">
+    <p class="p1">
+      {{d}} days, {{h}} hours, {{m}} minutes and {{s}} seconds until
```

Comment:
Bracketing the units so that changing the locale actually changes words. 👇 
```suggestion
      {{d}} {{days}}, {{h}} {{hours}}, {{m}} {{minutes}} and {{s}} {{seconds}} until
```